types/selenium-webdriver/test/chrome.ts

```diff
@@ -28,8 +28,8 @@ function TestChromeOptions() {
     options = options.androidProcess('com.android.chrome');
     options = options.androidUseRunningApp(true);
     options = options.setPerfLoggingPrefs({
-        enableNetwork: true, enablePage: true, enableTimeline: true,
```

Comment:
Please fix

Reply:
Hi @cnishina. My intention here was to test that a subset of these options are valid. Since Selenium and ChromeDriver both function fine without specifying all the options, I thought it was unnecessary for the types to require users to specify every option.

Further, the [ChromeDriver docs](https://chromedriver.chromium.org/capabilities#TOC-perfLoggingPrefs-object) don't mention a `enableTimeline` option. I don't feel comfortable removing the option entirely because I can't verify whether the option does something. It could be possible the ChromeDriver docs are out of date and missing the documentation for this option but again, I thought it would be useful to at least not require users to specify this option since it isn't documented and this test verifies that `setPerfLoggingPrefs()` accepts a Partial object.

If you'd prefer, I can leave this line as it is (with just the traceCategories fix) and add an additional line to verify that `setPerfLoggingPrefs` accepts a Partial object.

Reply:
It is an optional property on the interface, the original test appears to cover this. I don't think this matters either way.